cmd/nginx-ingress/aws.go

```diff
@@ -0,0 +1,108 @@
+// +build aws
+
+package main
+
+import (
+	"context"
+	"encoding/base64"
+	"errors"
+	"log"
+	"math/rand"
+
+	"github.com/aws/aws-sdk-go-v2/config"
+	"github.com/aws/aws-sdk-go-v2/service/marketplacemetering"
+	"github.com/aws/aws-sdk-go-v2/service/marketplacemetering/types"
+
+	"github.com/dgrijalva/jwt-go/v4"
+)
+
+const letterBytes = "abcdefghijklmnopqrstuvwxyzABCDEFGHIJKLMNOPQRSTUVWXYZ0123456789-"
+
+var (
+	productCode   string
+	pubKeyVersion int32 = 1
+	pubKeyString  string
+	nonce         string
+)
+
+func init() {
+	rand.Seed(jwt.Now().UnixNano())
```

Comment:
This is used for security purposes, so this needs to be secure. Can you double check this is the correct way to securely seed the rand num generator ?

Reply:
I'm pretty sure it was secure enough already, but I've updated it to use `crypto/rand` just in case 👍

Reply:
magic thanks!